### ISSUE TITLE
TunedTabPFNClassifier give up to fit by sklearns library warning

### ISSUE THREAD

**Issue by handsomeman118** [PriorLabs/tabpfn-extensions#91]
### Describe the bug

The model TunedTabPFNClassifier runing with roc_auc score.I find that the score computing could get waring when I run a binary classify task.The code like " y_pred = model.predict_proba(X_val) ; score = roc_auc_score(y_val, y_pred, multi_class="ovr")". API params get two shape like (sample,) and (sample, 2). Then the code blocket will give up and model can't fit.

### Steps/Code to Reproduce

clf = TunedTabPFNClassifier(n_trials=10, metric="roc_auc", random_state=42, device="cuda")
data = pandas.read_csv(".....")
X = data[features]
y = data["class"]
scalar = MinMaxScalar()
X_scalar = scalar.fit_transform(X)
clf.fit(X_scalar , y)    # y conclud {0， 1}  it is a binary task

### Expected Results

file in  tabpfn-extensions\src\tabpfn_extensions\hpo\tuned_tabpfn.py

following is my Solution：
               elif task_type in ["binary"]:
                    if self.metric == MetricType.ACCURACY:
                        score = model.score(X_val, y_val)
                    elif self.metric in [MetricType.ROC_AUC]:
                        y_pred = model.predict_proba(X_val)
                        score = roc_auc_score(
                            y_val, [pairs[1] for pairs in y_pred], multi_class="ovr")  # important porting to voild warning
                    elif self.metric == MetricType.F1:
                        y_pred = model.predict(X_val)
                        score = f1_score(y_val, y_pred, average="weighted")
                elif task_type in ["multiclass"]:
                    if self.metric == MetricType.ACCURACY:
                        score = model.score(X_val, y_val)
                    elif self.metric in [MetricType.ROC_AUC]:
                        y_pred = model.predict_proba(X_val)
                        score = roc_auc_score(y_val, y_pred, multi_class="ovr")
                    elif self.metric == MetricType.F1:
                        y_pred = model.predict(X_val)
                        score = f1_score(y_val, y_pred, average="weighted")

### Versions

```shell

```

**Comment by noahho** [comment]
Does this bug still exist with the newest version?
